packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -162,14 +160,20 @@ export const getCaretDownButtonStyles = memoizeFunction((
           borderColor: 'ButtonText',
           color: 'ButtonText',
           MsHighContrastAdjust: 'none'
+        },
+        'i': {

```

Comment:
I don't think you want do be doing generalized things like this!

Reply:
It's a little safer to say `& .ms-Icon`

`i` selector means "all I tags in the entire app." Because you don't have an ampersand, we wouldn't be able to differentiate this between a fully qualified global css rule vs one scoped to the element only.